spark/v3.3/spark/src/test/java/org/apache/iceberg/TestManifestFileSerialization.java

```diff
@@ -199,7 +199,7 @@ private ManifestFile writeManifest(DataFile... files) throws IOException {
     Assert.assertTrue(manifestFile.delete());
     OutputFile outputFile = FILE_IO.newOutputFile(manifestFile.getCanonicalPath());
 
-    ManifestWriter writer = ManifestFiles.write(SPEC, outputFile);
+    ManifestWriter<DataFile> writer = ManifestFiles.write(SPEC, outputFile);
```

Comment:
I think we are still missing in `TestForwardCompatibility#testSparkCanReadUnknownTransform()`

this is applicable to all 5 versions of spark.

Reply:
good catch, fixed those as well, thanks